src/CMakeLists.txt

```diff
@@ -47,9 +52,11 @@ link_libraries(${NETHOST_LIBRARY})
 
 if (WIN32)
   set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
-else()
+elseif(UNIX AND NOT APPLE)
   link_libraries(m)
   link_libraries(stdc++fs)
+else()
+  link_libraries(m)
```

Comment:
Apple includes `std::filesystem` in `libc++` so there's no external linking needed here.

Reply:
👍